src/analysis/functions.rs

```diff
@@ -264,6 +281,12 @@ fn analyze_function(
 
     if async && !commented && env.config.library_name != "Gio" {
         imports.add("gio_ffi", version);
+        imports.add("gio", version);
+    }
+
+    if async && !commented {
+        imports.add("futures_core", version);
```

Comment:
This probably needs to be conditional: `#[cfg(feature = "futures")]`.

Reply:
Yes but that's currently not supported by the `imports`. I hacked around it where the imports are written out to add that cfg directive

Reply:
FYI: https://github.com/sdroege/gio-rs/blob/886afc3d001ae10d8f2e39c56112575ff1a6a2f9/src/auto/file_i_o_stream.rs#L11, not sure if this already includes this hack or not.

Reply:
Ah my hack failed. Thanks! Will fix

Reply:
done